eventuals/expected.h

```diff
@@ -187,9 +187,31 @@ auto Expected(T t) {
 
 ////////////////////////////////////////////////////////////////////////
 
-template <typename T>
-auto Unexpected(T t) {
-  return _Expected<_Unexpected>(make_exception_ptr_or_forward(std::move(t)));
+template <typename Error>
+auto Unexpected(Error error) {
```

Comment:
If 'Error' is 'std::runtime_error', 'std::exception_ptr' would be created, should we propagate 'std::runtime_error' as it is?